CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+* The `-g` is now passed to `curl` when checking links, to disable globbing. ([#2839], [#2842])
```

Comment:
Reading this, I am not sure how it would affect me as a user. Perhaps we can expand on it a bit, like so?
```suggestion
* The `-g` is now passed to `curl` when checking links, to disable globbing, which could cause undesirable behavior when checking links containing characters like for example `[`, `{`, `&` or `?`. ([#2839], [#2842])
```